sxw_soilwat.c

```diff
@@ -222,14 +223,36 @@ static void _update_productivity(void) {
         }
     }
 
+    // Calculate the cover fraction of each SOILWAT2 vegetation type (sum = 1):
+    // here, approximate cover with the contribution of vegetation type biomass
+    // to total biomass
+    if (GT(totbmass, 0.)) {
+        v->veg[SW_TREES].cov.fCover = (vegTypeBiomass[0] / totbmass);
+        v->veg[SW_SHRUB].cov.fCover = (vegTypeBiomass[1] / totbmass);
+        v->veg[SW_GRASS].cov.fCover = (vegTypeBiomass[2] / totbmass);
+        v->veg[SW_FORBS].cov.fCover = (vegTypeBiomass[3] / totbmass);
```

Comment:
@dschlaep I could be wrong here, but are grasses and forbs switched? Should it be:

```
v->veg[SW_GRASS].cov.fCover = (vegTypeBiomass[3] / totbmass);
v->veg[SW_FORBS].cov.fCover = (vegTypeBiomass[2] / totbmass);
```
I know we resolved this in bugfix_336 but it seems important to do it here too so there are no merge issues?

Reply:
@kpalmqui I agree that they are switched. I understood that we are addressing this issue #348 (biomass transfer) and the issue #336 (incorrect vegetation indices) separately, hence I did not repeat the fixes that already exists https://github.com/DrylandEcology/STEPWAT2/pull/338.

This is also why I created a testing branch that combines fixes to both issues for easier evaluation (https://github.com/DrylandEcology/STEPWAT2/pull/338#issuecomment-510662339)

Reply:
@dschlaep ok sounds good